builder/rootfs/etc/confd/templates/check-repos

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
 
+set -x
+
 export ETCD=${ETCD:-$HOST:4001}
 
 cd $(dirname $0) # absolute path
 
-for repo in *.git
+for repo in $(ls | grep .git)
```

Comment:
Sorry--I saw that `ls` would return an error in that case but thought it was appropriate behavior, but that's really not an error case for this script.